lib/usermanager.js

```diff
@@ -53,6 +52,21 @@ exports = module.exports = {
     'DuplicateUserError': DuplicateUserError
   },
 
+  /**
+   * checks if a user is permitted the action on the resource
+   *
+   * @param {string} action
+   * @param {object} user
+   * @param {string} resource
+   * @param {callback} next (function (err, isAllowed))
+   */
+
+  hasPermission: function(action, user, resource, next) {
+    permissions = require('./permissions');
```

Comment:
You shouldn't do this:
https://github.com/alanjames1987/Node.js-Best-Practices#never-require-modules-inside-of-functions